cluster-autoscaler/integration/framework.go

```diff
@@ -54,6 +55,28 @@ func rotateLogFile(fileName string) (*os.File, error) {
 	return os.Create(fileName)
 }
 
+func (driver *Driver) taintOrUnTaintInitialNodes(toTaint bool) error {
```

Comment:
Shouldn't we have different methods for adding and removing taints? That way, we don't have to pass this boolean to determine if we are adding or removing them.

Reply:
addressed in [a8a7cef](https://github.com/gardener/autoscaler/pull/187/commits/a8a7cef0877b589973075d0e53b70f2974810ab9)